src/Components/Home.jsx

```diff
@@ -0,0 +1,13 @@
+import ArticlesList from "./ArticlesList"
+
+
+
+function Home() {
+  return (
+    <section>
+      <ArticlesList />
+    </section>
+  );
+}
+
```

Comment:
There's no need for the Home component if its job is just to render the Articles component. You can render the Articles component in your route directly